### ISSUE TITLE
no-var-before-return: false positive

### ISSUE THREAD

**Issue by ajafff** [ajafff/tslint-consistent-codestyle#13]
```ts
var foo = {
   fn() {console.log(foo);}
};
return foo;
```
